docs/releases/unreleased.md

```diff
@@ -9,6 +13,7 @@
 ## compose
 
 - Moved `utils.pure_inference_mode` to `compose.pure_inference_mode` and `utils.warm_up_mode` to `compose.warm_up_mode`.
+- Pipeline parts can now be accessed by integer positions as well as by name.
```

Comment:
This is very useful!

Reply:
Should we update the PR description, as it will become the squashed commit name?

Reply:
Good idea :)

Reply:
But not sure how to do that exactly; do you know how?

Reply:
My bad, I meant "the PR title". My brain was not working properly